tests/set-namespace/rolebindings/Kptfile

```diff
@@ -0,0 +1,9 @@
+apiVersion: kpt.dev/v1
+kind: Kptfile
+metadata:
+  name: example
+pipeline:
+  mutators:
+    - image: gcr.io/kpt-fn/set-namespace:unstable
```

Comment:
it will be good to have a test case that involves `ensure-name-substring` function in the pipeline and ensure this change works well with that too.

Reply:
Do you think that should be included in this test or as a separate test?

Reply:
Having a separate test case which explicitly tests the interplay of both the functions will be good.

Reply:
Done!